### DISCUSSION TITLE
Pixel 7 not able to sync to onedrive

### DISCUSSION THREAD

**Discussion by JeThKo** [remotely-save/remotely-save#395] [Q&A]
I am having trouble connecting my pixel 7 pro to the vault in my OneDrive which was backed up via my PC. I'm able to point obsidian to "internal storage" on my phone, but don't see any of the folders from my OneDrive in there. I am able to find the OneDrive folders using my phones "files" app which shows an "internal storage" and a "other storage", the later being the one that contains the OneDrive folders. But I'm unable to navigate obsidian to that section of storage.

**Comment by fyears** [comment]
by using this plugin, you don't need to set up a vault in your mobile-onedrive folder. Just set up a vault in anywhere you like , the plugin can upload your files onto onedrive **website**
